linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/configure_opensc_card_drivers/rule.yml

```diff
@@ -0,0 +1,45 @@
+documentation_complete: true
+
+prodtype: rhel7,fedora
+
+title: 'Configure opensc Smart Card Drivers'
+
```

Comment:
Will these rules be included in a profile?

Reply:
Yes. I believe there is an SSSD rule that needs to be added as well before adding to the profile(s). Creating a separate PR for profile changes.